src/ngx_http_lua_common.h

```diff
@@ -236,6 +240,16 @@ union ngx_http_lua_srv_conf_u {
 
 
 typedef struct {
+    ngx_int_t                is_handler_inline;
+    u_char                  *chunkname;
+    ngx_http_complex_value_t source;    /*  inline script/script
```

Comment:
Seems like we need more spaces between the type names and the field names. It's a bit too crowded I'm afraid :)